statsviz.go

```diff
@@ -82,7 +81,7 @@ func Register(mux *http.ServeMux, opts ...Option) error {
 // updates metrics data and provides two essential HTTP handlers:
 //   - the Index handler serves Statsviz user interface, allowing you to
 //     visualize runtime metrics on your browser.
-//   - The Ws handler establishes a WebSocket connection allowing the connected
+//   - The Ws handler establishes a data connection allowing the connected
```

Comment:
The `Ws` handler is used in examples and is part of the public API, but since `ws` means websockets and the communication won't be performed via websockets anymore, the Ws handler must be renamed to `SSE`. Removing it would be a breaking change however.

So, we should keep the `Ws` handler and mark is as deprecated, and make it an alias of the `SSE` handler.